charts/gardener/gardenlet/templates/clusterrole-gardenlet.yaml

```diff
@@ -115,7 +115,7 @@ rules:
   resources:
   - mutatingwebhookconfigurations
   resourceNames:
-  - vpa-webhook-config-seed
+  - zzz-vpa-webhook-config-seed
```

Comment:
Not really related to this pull request, but the block above this one already includes a superset of the permissions requested here. Is there a special reason for the redundancy?

Reply:
No idea, tbh. Maybe just for historical reasons? @rfranzke and @ialidzhikov did the PRs for these sections originally, I think.

Reply:
done.